bokehjs/test/integration/glyphs/rect.ts

```diff
@@ -27,4 +28,17 @@ describe("Rect glyph", () => {
 
     await display(row([make_plot("canvas"), make_plot("svg"), make_plot("webgl")]))
   })
+
+  it("should support adding new glyph", async() => {
+    const p = fig([200, 200], {
+      title: null,
+      toolbar_location: null,
```

Comment:
No need for both. `fig()` is there to create bare plots, while keeping the figure API.

Reply:
This was a copy and paste of
https://github.com/bokeh/bokeh/blob/0cadaff30bc976a6fca39c07b538e891dc76651f/bokehjs/test/integration/regressions.ts#L93-L97
I'll remove both. Do you want me to similarly clean the original?

Reply:
Sure.